docs(readme): add Sonā to the list of Web Perf services (#16282)
GoogleChrome/lighthouse: readme.md

* **[Guardius](https://guardius.io)** - Guardius is a DevOps and DevSecOps SaaS platform that integrates Lighthouse to deliver automated web performance analysis. It not only provides metrics evaluation and automatic scanning but also enables performance comparisons across different periods and ongoing observation over time. Additionally, Guardius offers predefined and customized alerts tailored to your specific requirements. A free version of Guardius is available for users to explore its features.

> * **[Sonā](https://getsona.io)** - Powered by Lighthouse amongst others, Sonā delivers in-depth insights into your website’s health. Track changes over time, share reports, and receive actionable recommendations to improve performance, accessibility, SEO, best practices, and security. Sonā helps you make your website healthy — try it now for **free**.

Reviewer: ```suggestion
* **[Sonā](https://getsona.io)** - Powered by Lighthouse amongst others, Sonā delivers in-depth insights into your website’s health. Track changes over time, share reports, and receive actionable recommendations to improve performance, accessibility, SEO, best practices, and security. Sonā is free during its beta period.
```

Author: I appreciate your feedback on the text, but there will always be a free tier of Sonā.

Reviewer: I understand, but the pricing FAQ doesn't mention that. I would prefer to just mention what is currently true about the product rather than speculate about future pricing models.
